### PR TITLE
Fix bugs related to console chat (#415)

### LINKED ISSUE #415: Slow UI re-appearance and blinking message decorators
After sending the password message, there is a significant delay  (3-4 seconds) and the messages just instantly appear (as opposed to animating up).

After this the message decorators disappear only to reappear again.

Also Console dapp icon in top right is missing until this message decorator 'flash'

### PR DESCRIPTION
**Pull request by rasom**
fixes #415

### CODE REVIEW

**Comment by alwx** [comment]
There is another problem on iOS.
The process of creating a new account is so fast that the "gimmie a sec" message appears after other 3 messages.

screen shot 2016-11-04 at 14 23 19

**Comment by rasom** [comment]
@jarradh, so lets skip it in ios then?

**Comment by jarradh** [comment]
thats a simulator so i don't believe it, also how is it even possible ?

**Comment by jarradh** [comment]
i thought we have vector clock and also message should be done before call no?

**Comment by rasom** [comment]
We have 400ms delay there, it is faster than 400ms. I can check this on the real device a bit later

**Comment by rasom** [comment]
Oh, ok i know how to fix this.

**Comment by jarradh** [comment]
okay so lets skip it if next messages appear before 400ms, regardless of platform

**Comment by rasom** [comment]
ok

**Comment by jarradh** [comment]
also we should probably hide the "phew! .. " text from the next message if so fast

File: src/status_im/chat/sign_up.cljs [hunk 1]
```diff
@@ -105,25 +105,35 @@
...
-              :to           "me"
-              :new?         false}])
+              :to           "me"}]))
+
+(defn passpharse-messages [mnemonic messages-count]
+  (dispatch [:received-message
+             {:message-id   (random/id)
+              :content      (if (> messages-count 3)
+                              (label :t/phew-here-is-your-passphrase)
+                              (label :t/here-is-your-passphrase))
```
Comment (alwx) [inline src/status_im/chat/sign_up.cljs]: But there can be a situation when you have more than three messages, but need to display `:t/here-is-your-passphrase` message without "Phew" at the beginning.
Not sure that it is important, but probably it is not the best way to determine which phrase to show.

**Comment by alwx** [comment]
Everything is fine, can be merged.
The only comment I have is above, but maybe it is better to skip it for now (or leave a TODO in code) and move forward.

**Pull request merged**